notebooks/source/lotka_volterra_multiple.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        progress_bar=False if "NUMPYRO_SPHINXBUILD" in os.environ else True,
</pre><p>This is unnecessary. You can set <code>progress_bar=True</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done.</p>